price_pusher/src/price-config.ts

```diff
@@ -47,3 +48,84 @@ export function readPriceConfigFile(path: string): PriceConfig[] {
     return priceConfig;
   });
 }
+
```

Comment:
Why did you move it here? I think maybe it's better to be on `controller.ts`

Reply:
it is very closely related to price config. 
if config changes we will need to update the should Update logic too. 
controller doesn't need to know anything related to priceConfig. 
I am thinking in this direction.